### PR TITLE
Enhancement/annotations

### PR DESCRIPTION
**Pull request by ghost**
Adds:
- tests for the annotation and annotation processor classes
- ability to select a GFF file to load from disk

Issue #140

### CODE REVIEW

**Comment by niekvanderlaan** [comment]
I checked your code and overall it seems to work. What I am still missing however, is the removal of the annotations highlight upon, for example, choosing to highlight another annotation. I think we have a few options here:
- Implement to remove the highlighting of annotations upon selecting another
- Give the user an option to deselect certain highlighting
- Perhaps a more advanced option, but give the user the option to choose colors upon highlighting an annotation. This way the user could easily review multiple annotations at a time. This would ofcourse also require a short overview of currently highlighted annotations.

Let me know what you think and ofcourse I'm ready to help!

**Comment by ghost** [comment]
Feature requests 1 and 2 are feasible within the current time span, the latter more advanced one is not. While feature requests are being implemented this PR will be closed.

File: src/main/java/application/controllers/MainController.java [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
+import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.geometry.Pos;
 import javafx.scene.image.ImageView;
 import javafx.scene.control.*;
 import javafx.scene.layout.*;
 
+import java.io.FileNotFoundException;
 import java.net.URL;
 import java.util.*;
```
Comment (tieswesterborg) [inline src/main/java/application/controllers/MainController.java:16]: Avoid using .*

Comment (ghost) [inline src/main/java/application/controllers/MainController.java:16]: Saves LOC and prevents large useless lists of unused imports on code change. Won't fix

**Comment by tieswesterborg** [comment]
Ready to merge

**Pull request merged**